base/exports.jl

```diff
@@ -762,6 +762,7 @@ export
     graphemes,
     hex,
     hex2bytes,
+    hex2bytes!,
```

Comment:
this will need to be added to the stdlib doc index somewhere to show up in the rendered docs https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#adding-a-new-docstring-to-base